packages/vm/tests/api/runBlockchain.spec.ts

```diff
@@ -148,18 +138,12 @@ tape('runBlockchain', (t) => {
     await blockchain.putBlock(b2)
     await blockchain.putBlock(b3)
 
-    let head = await blockchain.getHead()
-    st.deepEqual(head.hash(), b3.hash(), 'block3 should be the current head')
```

Comment:
Same here.

Reply:
@holgerd77 
What happened here is that one of the changes from getHead() to getIteratorHead() involved this:
```
Note that the function deviates from the old behavior and returns the genesis hash instead of the current head block if an iterator has not been run
```

While other tests could be adapted, this test is trying test whether an invalid block has been removed by checking the head before and after vm.runBlockchain().
However with the change in behavior, this will always return the genesis block before vm.runBlockchain() has been called.

so the same test passes if changed to:
```
    let head = await blockchain.getIteratorHead()
    st.deepEqual(head.hash(), genesisBlock.hash(), "...")
```

unless there is another way to `run the iterator` ?